docs/guide/index.md

```diff
@@ -107,7 +107,7 @@ You can specify additional CLI options like `--port` or `--https`. For a full li
 | `--threads` | Enable Threads (default: `true`) |
 | `--silent` | Silent console output from tests |
 | `--isolate` | Isolate environment for each test file (default: `true`) |
-| `--reporter <name>` | Select reporter: `default`, `verbose`, `dot`, `junit` or `json` |
+| `--reporter <name>` | Select reporter: `default`, `verbose`, `dot`, `junit`, `json`, or a path to a custom reporter (e.g. `/path/to/reporter.ts`, `@company/reporter`) |
```

Comment:
We also have `reporter` section in `config/index.md`. Would be great if you could also add this information there

Reply:
Ok I've added it